src/app/CASESessionManager.h

```diff
@@ -165,15 +173,18 @@ class CASESessionManager : public OperationalSessionReleaseDelegate, public Sess
 private:
     OperationalSessionSetup * FindExistingSessionSetup(const ScopedNodeId & peerId, bool forAddressUpdate = false) const;
 
-    Optional<SessionHandle> FindExistingSession(const ScopedNodeId & peerId) const;
+    Optional<SessionHandle> FindExistingSession(
+        const ScopedNodeId & peerId,
+        const TransportPayloadCapability transportPayloadCapability = TransportPayloadCapability::kMRPPayload) const;
 
     void FindOrEstablishSessionHelper(const ScopedNodeId & peerId, Callback::Callback<OnDeviceConnected> * onConnection,
                                       Callback::Callback<OnDeviceConnectionFailure> * onFailure,
                                       Callback::Callback<OperationalSessionSetup::OnSetupFailure> * onSetupFailure,
 #if CHIP_DEVICE_CONFIG_ENABLE_AUTOMATIC_CASE_RETRIES
                                       uint8_t attemptCount, Callback::Callback<OnDeviceConnectionRetry> * onRetry
 #endif
-    );
+                                      ,
```

Comment:
This looks like it will not compile when CHIP_DEVICE_CONFIG_ENABLE_AUTOMATIC_CASE_RETRIES is false, to me.